packages/visualizations/src/components/Map/types.ts

```diff
@@ -95,31 +95,14 @@ export interface ChoroplethDataValue {
     label?: string;
 }
 
-export enum ChoroplethShapeTypes {
-    GeoJson = 'geojson',
-    VectorTiles = 'vtiles',
-}
-
-/** `ChoroplethShapeValue` implementation based on a GeoJSON FeatureCollection.  */
-export interface ChoroplethShapeGeoJsonValue {
```

Comment:
In order to use it oustide, I feel like we still need to export a type representing this object. Something like:

``` TypeScript
export type ChoroplethShapeGeoJsonValue = FeatureCollection;
```

Reply:
`ChoroplethGeoJsonOptions` already defines that it expects a `FeatureCollection` for `shapes`, which seems fine to me: I think it's clearer to talk about a well-known thing like GeoJSON feature collection, than something named by us which is equivalent.

Reply:
Indeed, but `FeatureCollection ` comes from `@types/geojson` which may not be installed in projets that may use our SDK. Plus, I feel like we also need to export it with our naming convention. But I don't have a strong opinion on this subject.

Reply:
But if we create a new type that is based on `@types/geojson`'s `FeatureCollection`, the projects will still need to have the package to resolve the type anyway. But indeed that's something I never thought about, how does it work in that case... Maybe it just does complain about the missing package anyway. And maybe there's a way to directly inline and package all the types in the definitions that are built (the `*.d.ts` files). It's worth having a look at the best practices when we have the time.

Regardng our naming conventions, I feel like it's you're installing the SDK, and having a look at the expected parameters, having "`shapes: FeatureCollection`" tells you immediately what you need and makes it simple. I don't think there's an added value to add our own named version that requires the developer to click one step further and understand that we're expecting GeoJSON, since there's really no nuance to it, it's really just a FeatureCollection without any other requirement. But it's worth a deeper philosophical discussion on a wednesday morning :)

Reply:
I really prefer using `FeatureCollection` as it is. It's more generic, more explicit and it doesn't induce the idea that there might be a difference.
The package isn't a problem IMO, it's in the dev dependencies anyways and only will only be useful for devs. So it will be installed by `npm i`.